shoop/core/models/orders.py

```diff
@@ -214,9 +219,15 @@ class Order(models.Model):
     extra_data = JSONField(blank=True, null=True)
 
     # Money stuff
-    taxful_total_price = MoneyField(editable=False, verbose_name=_('grand total'), default=0)
-    taxless_total_price = MoneyField(editable=False, verbose_name=_('taxless total'), default=0)
-    display_currency = models.CharField(max_length=4, blank=True)
+    taxful_total_price = TaxfulPriceProperty('taxful_total_price_value', 'currency')
+    taxless_total_price = TaxlessPriceProperty('taxless_total_price_value', 'currency')
+
+    taxful_total_price_value = MoneyValueField(editable=False, verbose_name=_('grand total'), default=0)
+    taxless_total_price_value = MoneyValueField(editable=False, verbose_name=_('taxless total'), default=0)
+    currency = CurrencyField()
+    prices_include_tax = models.BooleanField()  # TODO: (TAX) Document Order.prices_include_tax
```

Comment:
I thought that would be misleading.. that's the reason for the todo note. ;-)

Order.prices_include_tax is for the OrderLines, since their values have to be stored pre-tax-calculation, we need the boolean so we know how to calculate the taxes for the lines (upwards or downwards from the values stored in the lines). Does that make any sense?